src/index.js

```diff
@@ -1,33 +1,133 @@
+/* eslint-disable no-loop-func */
+/* eslint-disable no-restricted-globals */
 import './styles/style.css';
 
-const todoList = [
-  {
-    discription: 'wash the dishes',
-    completed: false,
-    index: 0,
-  },
-  {
-    discription: 'complete To Do list project',
-    completed: false,
-    index: 1,
-  },
-  {
-    discription: 'implement project code review suggestions',
-    completed: false,
-    index: 2,
-  },
-];
+class TaskManager {
+  constructor(taskName) {
+    this.taskName = taskName;
+    this.completed = false;
+    this.itemIndex = 0;
```

Comment:
- [ ] All your items have an index value of 0. But that should not be so. Kindly note that each item index should correspond with its position in the array. So the first item should have an index of 1, the second item should have an index of 2, the third item index of 3, and so on. Kindly fix this.

  ![image](https://user-images.githubusercontent.com/54848363/206332456-90d687eb-95f6-44c5-aca5-750ddff8aab1.png)